modules/client/src/client-common/interfaces/plugin.ts

```diff
@@ -31,15 +30,32 @@ export interface IProposalSettings {
   duration: number;
 }
 
-export interface IPluginSettings {
-  /** Float: 0 to 1 */
-  minSupport: number;
-  /** Float: 0 to 1 */
-  minTurnout: number;
-  /** In seconds */
+export type VotingSettings = {
+  /* default is standard */
+  votingMode?: VotingMode;
+  /** Float between 0 and 1 */
+  supportThreshold: number;
+  /** Float between 0 and 1 */
+  minParticipation: number;
   minDuration: number;
+  /* default is 0 */
+  minProposerVotingPower?: bigint;
+};
+
+export enum VotingMode {
+  STANDARD = "Standard",
+  EARLY_EXECUTION = "EarlyExecution",
+  VOTE_REPLACEMENT = "VoteReplacement",
 }
```

Comment:
Isn't this redundant with what is added in #129?

Reply:
#129 was merged into this branch not into `develop` thats why it is here again